自作例外のコンストラクタ定義不足でMSVCが警告を吐く

以前 (unique\_ptr の文脈で)「この警告にはほぼ意味がない」とのことでしたが、一応警告は警告なので対処お願いします
この際警告の握り潰しという対処方針でもいいかもしれません
```
angband-exceptions.h(8,1): warning C4623: 'SaveDataNotSupportedException': 既定のコンストラクターは暗黙的に削除済みとして定義されました
```

--- a/src/system/angband-exceptions.h
+++ b/src/system/angband-exceptions.h
@@ -4,5 +4,6 @@
 
 class SaveDataNotSupportedException : public std::runtime_error {
 public:
+    SaveDataNotSupportedException() = delete;
     using std::runtime_error::runtime_error;
 };

